ops/Jenkinsfile.build_trigger

```diff
@@ -0,0 +1,60 @@
+DEPLOY = 'Do Not Deploy'
+
+pipeline {
+  agent {
+    node {
+      label ''
+      customWorkspace 'workspace/bcda-static-site-build_trigger'
+    }
+  }
+  triggers {
+     pollSCM 'H/2 * * * *'
+     cron 'H 7 * * 1-5'
```

Comment:
Is this `cron` necessary?  I would think the `pollSCM` would be sufficient.  Unlike the API, we probably don't need to test the entire deploy pipeline daily for our `stage` env.